### PR TITLE
Protectli vault cml/rel 1.0.16

### PR DESCRIPTION
**Pull request by mkopec**

### CODE REVIEW

**Comment by mkopec** [comment]
Reverted back to LPC discrete TPM as discussed

**Comment by mkopec** [comment]
Reverted the blobs URL to the previous one

File: src/security/lockdown/lockdown.c [hunk 1]
```diff
@@ -3,8 +3,42 @@
...
+	uint8_t var;
+	uint32_t size;
+
+	if (smmstore_lookup_region(&rdev))
+		return false;
+
+	size = sizeof(var);
+	ret = efi_fv_get_option(&rdev, &dasharo_system_features_guid, "LockBios", &var, &size);
+	if (ret != CB_SUCCESS)
+		return true;
```
Comment (mkopec) [inline src/security/lockdown/lockdown.c:34]: What should be the default here? The default value of `BIOS Lock` in the setup menu is disabled, and it seems that the variable is created only after first toggling it. We enable bios lock here if it doesn't exist, so tianocore displays a wrong value by default.

File: dasharo/releases.yaml [hunk 2]
```diff
@@ -1,4 +1,17 @@
 releases:
+        - version: v1.0.16
+          added:
+                - Vboot Verified + Measured Boot
```
Comment (miczyg1) [inline dasharo/releases.yaml:4]: measured boot and verified boot are separate features

File: dasharo/releases.yaml [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+                - Vboot recovery notification in TianoCore
+                - UEFI Shell
+                - UEFI Secure Boot
+                - Intel PTT Firmware TPM support
```
Comment (miczyg1) [inline dasharo/releases.yaml:8]: We keep LPC TPM for now, remove

File: dasharo/releases.yaml [hunk 4]
```diff
@@ -1,4 +1,17 @@
 releases:
+        - version: v1.0.16
+          added:
+                - Vboot Verified + Measured Boot
+                - Vboot recovery notification in TianoCore
```
Comment (miczyg1) [inline dasharo/releases.yaml:5]:
```suggestion
                - Vboot recovery notification in UEFI Payload
```

File: dasharo/releases.yaml [hunk 5]
```diff
@@ -1,4 +1,17 @@
...
+                - UEFI Shell
+                - UEFI Secure Boot
+                - Intel PTT Firmware TPM support
+          changed:
+                - FSP changed to CometLake2
+          known_issues:
+                - description: i225 network controller initialization takes too much time
```
Comment (miczyg1) [inline dasharo/releases.yaml:12]: We can easily fix it, see PROT-462 @macpijan

File: src/security/lockdown/lockdown.c (hunk 1 above)
Comment (miczyg1) [reply src/security/lockdown/lockdown.c:34]: We should probably initialize it correctly in TianoCore. @SergiiDmytruk WDYT?

**Review by miczyg1 — CHANGES_REQUESTED** [review]
@mkopec Please bump version in https://github.com/Dasharo/coreboot/blob/protectli_vault_cml/release/.coreboot-version file

File: dasharo/releases.yaml (hunk 3 above)
Comment (mkopec) [reply dasharo/releases.yaml]: Removed

File: dasharo/releases.yaml (hunk 4 above)
Comment (mkopec) [reply dasharo/releases.yaml]: Done

**Comment by mkopec** [comment]
> @mkopec Please bump version in https://github.com/Dasharo/coreboot/blob/protectli_vault_cml/release/.coreboot-version file

Done

File: src/security/lockdown/lockdown.c (hunk 1 above)
Comment (SergiiDmytruk) [reply src/security/lockdown/lockdown.c:34]: Right. Always initializing is better than relying on defaults matching each other.

Comment (mkopec) [reply src/security/lockdown/lockdown.c:34]: Related PR: https://github.com/Dasharo/DasharoModulePkg/pull/3

File: dasharo/releases.yaml (hunk 5 above)
Comment (macpijan) [reply dasharo/releases.yaml:12]: Thanks for reminding

**Comment by mkopec** [comment]
Rebased on top of the latest `protectli_vault_cml/develop`.

File: dasharo/releases.yaml (hunk 5 above)
Comment (mkopec) [reply dasharo/releases.yaml:12]: Fixed, moved this issue to the `fixed` section

**Comment by mkopec** [comment]
@miczyg1 All the new features are merged, i think this is ready for review

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**